OpenOversight/app/mail_client.py

```diff
@@ -0,0 +1,76 @@
+"""This file houses the classes used to send emails."""
+import base64
+from email.mime.text import MIMEText
+
+from apiclient import errors
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+
+
+DEFAULT_SENDER_ADDRESS = "chi-oo@lucyparsonslabs.com"
+
+
+class Email:
+    def __init__(self, body: str, subject: str, receiver: str):
+        self.body = body
+        self.receiver = receiver
+        self.sender = DEFAULT_SENDER_ADDRESS
+        self.subject = subject
+
+    @classmethod
+    def create_message(cls):
+        message = MIMEText(cls.body)
+        message["to"] = cls.receiver
+        message["from"] = cls.sender
+        message["subject"] = cls.subject
+        return {"raw": base64.urlsafe_b64encode(message.as_string())}
+
+
+class ConfirmEmailAddressEmail(Email):
+    def __init__(self, receiver: str):
+        subject = "CONFIRM EMAIL ADDRESS TITLE"
+        body = "CONFIRM EMAIL ADDRESS BODY"
+        super.__init__(body, subject, receiver)
+
+
+class ChangeEmailAddressEmail(Email):
+    def __init__(self, receiver: str):
+        body = "CONFIRM EMAIL ADDRESS BODY"
+        subject = "CHANGE EMAIL ADDRESS TITLE"
+        super.__init__(body, subject, receiver)
+
+
+class ChangePasswordEmail(Email):
+    def __init__(self, receiver: str):
+        body = "CHANGE PASSWORD BODY"
+        subject = "CHANGE PASSWORD TITLE"
+        super.__init__(body, subject, receiver)
```

Comment:
I'm hoping this inherited object structure will give us some flexibility when it comes to multiple email types.